tensorflow/core/api_def/base_api/api_def_Acosh.pbtxt

```diff
@@ -3,9 +3,10 @@ op {
   summary: "Computes inverse hyperbolic cosine of x element-wise."
   description: <<END
 Given an input tensor, the function computes inverse hyperbolic cosine of every element.
-Input range is [1, infinity). It returns `nan` if the input lies outside the range.
+Input range is `[1, inf)`. It returns `nan` if the input lies outside the range.
```

Comment:
If the output below is correct, then the input range is `[1, inf]`, since inf -> inf, not inf -> nan.